docs/dev/create_a_tile.md

```diff
@@ -0,0 +1,321 @@
+ddev config set repo extras
+{{< /code-block >}}
+
+## Populate the integration tile scaffolding
+
+For Datadog API integrations that will be available out-of-the-box on the [Integrations page][102], use the Datadog Development Toolkit to create scaffolding for a tile-only listing.
+
+1. Make sure you're inside the `integrations-extras` directory:
+
+   ```shell
+   cd $HOME/dd/integrations-extras
+   ```
+
+1. Run the `ddev` command with the `-t tile` option:
+
+   ```shell
+   ddev create -t tile "<Offering Name>"
+   ```
+
+[101]: https://docs.datadoghq.com/developers/integrations/python
+[102]: https://github.com/Datadog/integrations-extras
+[103]: https://app.datadoghq.com/integrations
+
+{{% /tab %}}
+
+{{% tab "Build a tile on the Marketplace" %}}
+
+{{< img src="developers/integrations/marketplace_tile.png" alt="A tile representing an example offering on the Marketplace page" style="width:30%" >}}
+
+To build a tile on the [**Marketplace** page][104]: 
+
+<div class="alert alert-warning">If you have already gone through the steps to create an Agent integration and have built out a scaffolding, you can skip directly to <a href="https://docs.datadoghq.com/developers/integrations/create_a_tile/#complete-the-necessary-integration-asset-files">completing the necessary integration asset files</a>. 
+</div>
+
+1. See [Build a Marketplace Offering][102] to request access to the [Marketplace repository][101].
+
+1. Create a `dd` directory:
+   
+   ```shell
+   mkdir $HOME/dd
+   ```
+
+   The Datadog Development Toolkit command expects you to be working in the `$HOME/dd/` directory. This is not mandatory, but working in a different directory requires additional configuration steps.
+
+1. Once you have been granted access to the Marketplace repository, create the `dd` directory and clone the `marketplace` repository:
+   
+   ```shell
+   git clone git@github.com:DataDog/marketplace.git
+   ```
+
+1. Create a feature branch to work in:
+
+    ```shell
+    git switch -c <YOUR INTEGRATION NAME> origin/master
+    ```
+
+## Install and configure the Datadog development toolkit
+
+The Agent Integration Developer Tool allows you to create scaffolding when you are developing an integration by generating a skeleton of your integration tile's assets and metadata. For instructions on installing the tool, see [Install the Datadog Agent Integration Developer Tool][103].
+
+Once you have installed the Agent Integration Developer Tool, configure it for the `marketplace` repository:
+
+Set `marketplace` as the default working repository:
+
+{{< code-block lang="shell" >}}
+ddev config set marketplace $HOME/dd/marketplace
+ddev config set repo marketplace
+{{< /code-block >}}
+
+If you used a directory other than `$HOME/dd` to clone the `marketplace` directory, use the following command to set your working repository:
+
+{{< code-block lang="shell" >}}
+ddev config set marketplace <PATH/TO/MARKETPLACE>
+ddev config set repo marketplace
+{{< /code-block >}}
```

Comment:
~~~suggestion
```shell
ddev config set marketplace $HOME/dd/marketplace
ddev config set repo marketplace
```

If you used a directory other than `$HOME/dd` to clone the `marketplace` directory, use the following command to set your working repository:

```shell
ddev config set marketplace <PATH/TO/MARKETPLACE>
ddev config set repo marketplace
```
~~~